apps/prairielearn/src/components/Scorebar.html.ts

```diff
@@ -15,13 +15,13 @@ export function Scorebar(
       style="min-width: ${minWidth}; max-width: ${maxWidth};"
     >
       <div class="progress-bar bg-success" style="width: ${Math.floor(Math.min(100, score))}%">
-        ${score >= 50 ? `${Math.floor(score)}%` : ''}
+        ${score >= 50 ? `${Math.round(score * 100) / 100}%` : ''}
```

Comment:
We'd still want to use `Math.floor` here. We _never_ want to give students the impression that they got more points than they actually did. For instance, if their actual score was 21.1777777%, we don't want to show 21.18%, as that would be a higher number than what they actually earned.

I think 2 decimal points is reasonable, as that's the same as when we show points information:

![Screenshot 2024-11-07 at 13 10 25](https://github.com/user-attachments/assets/9370b9a2-18aa-459f-87e4-3a960f11e915)

As long as we're here: consider lifting this computation up to a variable, like:

```js
const percentage = `${Math.round(score * 100) / 100}%`;
```

Reply:
Good point - reverting back to floor now and moving it to a variable